x/twap/store_test.go

```diff
@@ -475,3 +480,48 @@ func (s *TestSuite) TestGetAllHistoricalPoolIndexedTWAPs() {
 		})
 	}
 }
+
+func (s *TestSuite) TestAccumulatorOverflow() {
+	maxSpotPrice := gammtypes.MaxSpotPrice
+	tests := map[string]struct {
+		// timeDelta is duration in nano seconds.
```

Comment:
Isn't timedelta in milliseconds now as per this issue(https://github.com/osmosis-labs/osmosis/issues/2209)?